site_gen.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+CLUSTER_NAME=$1
+profile=$2
+RAN_MANIFEST_REPO=$3
+ORG_RAN_MANIFEST_REPO=https://github.com/redhat-ztp/ztp-ran-manifests
+
+if [ $profile == "cu" ]; then
+  cp -rf subscriptions/sites/sample_site_cu_subscription subscriptions/sites/${CLUSTER_NAME}_subscription
+  sed -i "s/sample-site-cu/$CLUSTER_NAME/g" subscriptions/sites/${CLUSTER_NAME}_subscription/*
+elif [ $profile == "du" ]; then
+  cp -rf subscriptions/sites/sample_site_du_subscription subscriptions/sites/${CLUSTER_NAME}_subscription
```

Comment:
why not just replace the var? sample_site_${profile}  instead of the if... this will allow to add more profiles in the future

Reply:
just more restrict check for cu and du profile name and gives more flexibility for directory name